docs/quality-assurance/Web-Accessibility.md

```diff
@@ -0,0 +1,26 @@
+# Web Accessibility Testing
+
+Accessibility is not a 'nice to have', it's a legal requirement for companies and organisations to be inclusive for everyone.
+
+Accessibility Testing is not confirmation-based. Instead of testing the functionality works, testers need to look for how it **can't** be used because of an accessibility barrier.
+
+Automated tools include:
+
+- core installed in delivery pipeline - https://www.deque.com/axe/
+- Lighthouse - https://developers.google.com/web/tools/lighthouse
+
+Accessibility Testing is 'hands on'. While automated tests will catch things like missing alt text, contrast problems, and missing form labels; they are no substitute for manual hands on testing. Pre-written manual testing scripts cannot be relied upon because they provide no more insight than automated testing. The tester must explore different ways of using the application in order to tease out problems with accessibility.
```

Comment:
Can't they? A "script" listing all the areas that can be verified with manual effort would be very useful.